scripts/geocoding.py

```diff
@@ -9,8 +9,8 @@
 
 
 dirname = os.path.dirname(__file__)
-input   = os.path.join(dirname, '../../reporting/collections-metadata.tsv')
-output  = os.path.join(dirname, '../target/coords.csv')
+input   = os.path.join(dirname, 'reporting/collections-metadata.tsv')
```

Comment:
See comment above. It would be better to access the required input files from a single point e.g. `target/`. It can be copied there before running the docker container.